mantidproject/mantid
# Indirect - ConvFit - Loading res before red causes error

**Description of work.**
This PR fixes an out of range error message when the resolution file is loaded in `ConvFit` before the reduced file.

**To test:**
1.`Interfaces`->`Indirect`->`Data Analysis`->`ConvFit` Tab
2. Load resolution file below
3. There should be no error!
4. Load the reduced file below and make sure it runs without a hitch. Fit type `One lorentzian`

**Data files**
Reduced: [irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2496249/irs26176_graphite002_red.zip)
Resolution: [irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2496250/irs26173_graphite002_res.zip)


Fixes #23863

I don't think this needs release notes because it is a small fix?

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
